rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json

```diff
@@ -0,0 +1,646 @@
+                                                        "primary_airflow": 471.94744319999984,
+                                                        "supply_design_heating_setpoint_temperature": 29.444444444444457
+                                                    }
+                                                ],
+                                                "design_thermostat_heating_setpoint": 18.33333333333337,
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "lighting_space_type": "STAIRWELL"
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilating_air_conditioning_systems": [
+                                            {
+                                                "id": "System 10",
+                                                "heating_system": {
+                                                    "id": "Heating Coil 1",
+                                                    "type": "FURNACE",
+                                                    "energy_source_type": "ELECTRICITY"
+                                                },
+                                                "fan_system": {
+                                                    "id": "CAV Fan System 1",
+                                                    "fan_control": "CONSTANT",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 75,
+                                                            "design_airflow": 117.98686079999996,
+                                                            "is_airflow_autosized": true
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 25
+                                                        }
+                                                    ],
+                                                    "minimum_outdoor_airflow": 0.0
```

Comment:
To fail, this ```minimum_outdoor_airflow``` value must be greater than the ```design_airflow``` under the ```supply_fans```. Could you check this again?